test/lib/processors/minifier.js

```diff
@@ -200,10 +206,84 @@ test();`;
 
 	const expected = `//@ui5-bundle sap/ui/my/module.js
 function test(t){var o=t;console.log(o)}test();
-//# sourceMappingURL=test.js.map`;
+${SOURCE_MAPPING_URL}=test.js.map`;
+	t.deepEqual(await resource.getString(), expected, "Correct minified content");
+});
+
+test("addSourceMappingUrl=false", async (t) => {
+	const content = `
+//@ui5-bundle sap/ui/my/module.js
```

Comment:
should we add there some indentation?

Reply:
This is by intention as the file content should not have any leading tabs or spaces.

Reply:
Thx